adele-payment-manager/src/main/java/com/microservicesteam/adele/payment/PaypalObjectFactory.java

```diff
@@ -0,0 +1,22 @@
+package com.microservicesteam.adele.payment;
+
+import com.paypal.api.payments.Payment;
+import com.paypal.base.rest.APIContext;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@AllArgsConstructor
+public class PaypalObjectFactory {
```

Comment:
I suggest to create a Proxy class for PayPal methods, let me show it. Shout if you don't like it.

Reply:
See #73